folders/backend/express-postgres/server.ts

```diff
@@ -0,0 +1,109 @@
+import express from 'express';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+import pg from 'pg';
+
+pg.types.setTypeParser(pg.types.builtins.NUMERIC, parseFloat);
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: { rejectUnauthorized: false },
+});
+
+const app = express();
+
+app.get('/api/actors/:actorId', async (req, res, next) => {
+  try {
+    const { actorId } = req.params;
+    if (actorId === undefined) {
+      throw new ClientError(400, 'actorId is required');
+    }
+    const sql = `
+      select
+        "actorId",
+        "firstName",
+        "lastName"
+      from "actors"
+      where "actorId" = $1;
+    `;
+    const params = [actorId];
+    const result = await db.query(sql, params);
+    const actor = result.rows[0];
+    if (!actor) {
+      throw new ClientError(404, `actor ${actorId} not found`);
+    }
+    res.send(actor);
+  } catch (err) {
+    next(err);
+  }
+});
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+      SELECT "filmId", "title", "replacementCost"
```

Comment:
It's easiest to just use `*`. In an API, that's almost always what you will want.